smoke-test/tests/cypress/cypress/e2e/mutations/managed_ingestion.js

```diff
@@ -31,8 +31,7 @@ describe("run managed ingestion", () => {
         cy.waitTextVisible(testName)
 
         cy.contains(testName).parent().within(() => {
-            // TODO: Skipping until disk size resolved
-            // cy.contains("Succeeded", {timeout: 30000})
+            cy.contains("Succeeded", {timeout: 1200000})
```

Comment:
is this timeout reasonable? the units are milliseconds right, so this is 20 minutes?